src/Simulation/EntryPointDriver.Tests/Tests.fs

```diff
@@ -989,6 +989,42 @@ let ``Uses documentation`` () =
     given ["-h"] |> yields message
     given ["-?"] |> yields message
 
+[<Fact>]
+let ``Shows help text for generateazurepayload command`` () =
+    let name = Path.GetFileNameWithoutExtension (Assembly.GetEntryAssembly().Location)
+    let message =
+        name
+        |> sprintf "Usage:
+                      %s generateazurepayload [options]
+
+                    Options:
+                      -n <n> (REQUIRED)                                   A number.
+                      --pauli <PauliI|PauliX|PauliY|PauliZ> (REQUIRED)    The name of a Pauli matrix.
+                      --my-cool-bool (REQUIRED)                           A neat bit.
```

Comment:
The fragment `--my-cool-bool (REQUIRED)` makes an impression that the flag `--my-cool-bool` (without any value following it) always has to be specified in the command line. Why one needs a flag that has to be specified _always_?

Reply:
In this case, `--my-cool-bool` comes from the [Help](https://github.com/microsoft/qsharp-runtime/blob/f95b1d4c338234be35cdced88e8f9e6be98e9477/src/Simulation/EntryPointDriver.Tests/Tests.qs#L361) Q# test case, and all arguments for entry-point operations translate into required command line options for standalone executables.

Reply:
I was expecting something like 
> --my-cool-bool **<false|true>** (REQUIRED)

rather than
> --my-cool-bool (REQUIRED)

Reply:
This is a good point! I created #824 to improve this in a separate PR since it is out of the scope of this one.